stores/yaml/store_test.go

```diff
@@ -281,3 +281,42 @@ func TestComment7(t *testing.T) {
 	assert.Equal(t, string(COMMENT_7_OUT), string(bytes))
 	assert.Equal(t, COMMENT_7_OUT, bytes)
 }
+func TestDuplicateAttributes(t *testing.T) {
+	// Duplicate keys are _not_ valid yaml.
+	//
+	// See https://yaml.org/spec/1.2.2/
+	// > The content of a mapping node is an unordered set of key/value node pairs,
+	// > with the restriction that each of the keys is unique.
+	//
+	data := `
+hello: Sops config file
+hello: Duplicates are not ok
+rootunique:
+  key2: "value"
+  key2: "foo"
+`
+	s := new(Store)
+	_, err := s.LoadPlainFile([]byte(data))
+	if err == nil {
+		t.Errorf("Expected error")
+	}
```

Comment:
Why not use [NotNil](https://pkg.go.dev/github.com/stretchr/testify/assert#NotNil) here?
```suggestion
	assert.NotNil(err)
```
(Same below.)